### PR TITLE
Update submit_debug_info.rst

### PR DESCRIPTION
**Pull request by valarmathip**
Updated the submit_debug_info.rst based on the PR https://github.com/StackStorm/st2/pull/2355

### CODE REVIEW

**Comment by manasdk** [comment]
Travis is unhappy

Warning, treated as error:
/home/travis/build/StackStorm/st2docs/docs/source/troubleshooting/submit_debug_info.rst:117: ERROR: Unexpected indentation.
make: *** [.docs] Error 1

File: docs/source/troubleshooting/submit_debug_info.rst [hunk 1]
```diff
@@ -81,3 +81,125 @@ and uploaded to StackStorm.
...
+* log_file_paths - an additional set of log files to gather
+* conf_file_paths - an additional set of config files to gather
+* s3_bucket - the S3 bucket to upload the archive to
+* gpg - gpg key and fingerprint to use when encrypting the archive
+* shell_commands - a list of shell commands to execute and capture the output from
+* company_name - the company name to show in the interactive prompts and log messages
+
+Sample Config yaml file:
+
+log_file_paths:
```
Comment (manasdk) [inline docs/source/troubleshooting/submit_debug_info.rst]: use .. code-block:: yaml or .. sourcecode:: yaml before the yaml block and indent everything yaml with 1tab/4spaces.

Comment (valarmathip) [inline docs/source/troubleshooting/submit_debug_info.rst]: thanks @manasdk

File: docs/source/troubleshooting/submit_debug_info.rst [hunk 2]
```diff
@@ -81,3 +81,128 @@ and uploaded to StackStorm.
     2015-02-10 17:43:49,016  INFO - Collecting files...
     2015-02-10 17:43:49,770  INFO - Creating tarball...
     2015-02-10 17:43:49,912  INFO - Debug tarball successfully generated and can be reviewed at: /tmp/st2-debug-output-vagrant-ubuntu-trusty-64-2015-02-10-17:43:49.tar.gz
+
+
+Submitting debugging information to Plexxi
```
Comment (andrew-regan) [inline docs/source/troubleshooting/submit_debug_info.rst]: Probably want to make this more generic and not talk about Plexxi. 
"Customizing the debug information gathered"

File: docs/source/troubleshooting/submit_debug_info.rst [hunk 3]
```diff
@@ -81,3 +81,128 @@ and uploaded to StackStorm.
...
+==========================================
+
+The enhancement work will allow submit_debug_info.py to be customized for specific deployments (Plexxi's as an example :-) by loading a set of overrides from a YAML file. The following config options can now be specified:
```
Comment (andrew-regan) [inline docs/source/troubleshooting/submit_debug_info.rst]: "submit_debug_info.py can be customized for specific deployments by loading a set of overrides from a YAML file. The following config options can now be specified:"

File: docs/source/troubleshooting/submit_debug_info.rst [hunk 4]
```diff
@@ -81,3 +81,128 @@ and uploaded to StackStorm.
...
+        salt_log_files_path: /var/log/*salt.log
+        mongodb_log_files_path: /var/log/mongod/*
+        nginx_log_files_path: /var/log/nginx/*
+        yum_log_files_path: /var/log/yum.log
+    conf_file_paths:
+        st2_config_file_path: /etc/st2/st2.conf
+        mistral_config_file_path: /etc/mistral/mistral.conf
+    s3_bucket:
+        url: https://plexxi-support.s3.amazonaws.com/
+    gpg:
```
Comment (andrew-regan) [inline docs/source/troubleshooting/submit_debug_info.rst]: Can you switch this over to the StackStorm GPG finger print and key.

File: docs/source/troubleshooting/submit_debug_info.rst [hunk 5]
```diff
@@ -81,3 +81,128 @@ and uploaded to StackStorm.
...
+        message_files_path: /var/log/messages*
+        salt_log_files_path: /var/log/*salt.log
+        mongodb_log_files_path: /var/log/mongod/*
+        nginx_log_files_path: /var/log/nginx/*
+        yum_log_files_path: /var/log/yum.log
+    conf_file_paths:
+        st2_config_file_path: /etc/st2/st2.conf
+        mistral_config_file_path: /etc/mistral/mistral.conf
+    s3_bucket:
+        url: https://plexxi-support.s3.amazonaws.com/
```
Comment (andrew-regan) [inline docs/source/troubleshooting/submit_debug_info.rst]: Can you switch this over to the StackStorm S3 bucket.

File: docs/source/troubleshooting/submit_debug_info.rst [hunk 6]
```diff
@@ -81,3 +81,128 @@ and uploaded to StackStorm.
...
+
+.. sourcecode:: yaml
+
+    ---
+    log_file_paths: 
+        st2_log_files_path: /var/log/st2/*.log
+        mistral_log_files_path: /var/log/mistral*.log
+        rabbitmq_log_files_path: /var/log/rabbitmq/*
+        message_files_path: /var/log/messages*
+        salt_log_files_path: /var/log/*salt.log
```
Comment (andrew-regan) [inline docs/source/troubleshooting/submit_debug_info.rst]: Delete

File: docs/source/troubleshooting/submit_debug_info.rst [hunk 7]
```diff
@@ -81,3 +81,128 @@ and uploaded to StackStorm.
...
+        mongodb_log_files_path: /var/log/mongod/*
+        nginx_log_files_path: /var/log/nginx/*
```
Comment (andrew-regan) [inline docs/source/troubleshooting/submit_debug_info.rst]: Delete

File: docs/source/troubleshooting/submit_debug_info.rst [hunk 8]
```diff
@@ -81,3 +81,128 @@ and uploaded to StackStorm.
...
+            hEEHYx/KajN0vCpQkKa3lWzviszak0nLs1TO8ppNPHgv3otNVFhLWZiIzv/A4LTY
+            WwShxYmHXZfu8p4wVeYNhT/g0ThFz8QpnzpZBO8PfROTS/5/v7aJrmkUTY/tTDhb
+            VDEaUjnGCtgflosRXd/uUK+VP0FibjJnP+W93PThnkpTJmFCuy5WPJlBYPHAMmZF
+            Tg==
+            =qDpc
+            -----END PGP PUBLIC KEY BLOCK-----
+    shell_commands:
+        cmd1: rpm -qa
+    company_name:
+        name: Plexxi
```
Comment (andrew-regan) [inline docs/source/troubleshooting/submit_debug_info.rst]: name: MyCompany

File: docs/source/troubleshooting/submit_debug_info.rst [hunk 9]
```diff
@@ -81,3 +81,128 @@ and uploaded to StackStorm.
...
+
+To send debug information to Plexxi, simply invoke the command shown below:
+
+.. sourcecode:: bash
+
+    submit-debug-info.sh
+
+    This will submit the following information to Plexxi: logs, configs, content, system_info, shell_commands
```
Comment (andrew-regan) [inline docs/source/troubleshooting/submit_debug_info.rst]: Change all occurrences of Plexxi to MyCompany

**Comment by dzimine** [comment]
The bug you see that breaks the check is already fixed in master, if you merge with master it should pass (unless there're other warnings).

**Comment by andrew-regan** [comment]
Thanks. Let me try to update.

**Comment by andrew-regan** [comment]
This can be closed.

**Comment by manasdk** [comment]
I think https://github.com/StackStorm/st2docs/pull/52 has what we needed.

**Pull request closed without merging**